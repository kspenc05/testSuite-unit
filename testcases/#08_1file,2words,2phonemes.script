#!/bin/sh
./cmulookup cmudict-0.7b input/2words.txt