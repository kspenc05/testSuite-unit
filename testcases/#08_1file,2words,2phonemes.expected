G R IY1 N
G L UW1
=====
DONE! (2 words)
